float64[9] joint_pos
float64[9] joint_spd
float64[6] eff_pose
float64[6] eff_spd
float64 timestamp
